.github/workflows/manylinux/entrypoint.sh

```diff
@@ -76,4 +85,4 @@ fi
 rm -f dist/*-linux*.whl
 
 echo "Succesfully build wheels:"
-find . -type f -iname "*-manylinux*.whl"
\ No newline at end of file
+find . -type f -iname "*-manylinux*.whl"
```

Comment:
I thought the wheel name for aarch64 would be something else other than -manylinux? Also L85 might be not needed for aarch64

Reply:
It is required as before auditwheel the aarch64 wheels will be generated with name "onnx-1.8.1-cp39-cp39-linux_aarch64.whl" and  auditwheel repair creates new wheel  file with name "onnx-1.8.1-cp39-cp39-manylinux2014_aarch64.whl".